frontend/src/views/RunView.vue

```diff
@@ -161,11 +158,16 @@ onMounted(() => {
     </aside>
   </div>
   <div v-else>
-    <p>QC data will appear here</p>
+    <p>Well QC data will appear here</p>
   </div>
 </template>
 
 <style>
+
+#run_wells {
+  width: 100%;
```

Comment:
Looks goofy at 3440x1440, but I think that's not a problem on Sanger hardware...